partner_multi_relation/models/res_partner.py

```diff
@@ -171,3 +171,23 @@ def get_partner_type(self):
         """
         self.ensure_one()
         return 'c' if self.is_company else 'p'
+
+    @api.multi
+    def action_view_relations(self):
+        for contact in self:
+            relation_ids = self.env['res.partner.relation.all'].\
```

Comment:
For style I would like to avoid the '\' line continuation, where not really needed.
If you assign `relation_model = 'self.env['res.partner.relation.all'] and put if for the loop, you can use
`relation_model.search(` and the can put the conditons on the next lines indented by 4 spaces.